src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colour.values().length);
+        return (Colour.values()[index]).toString();
```

Comment:
you need to add `toLowerCase()` here

`toString()` will return you `WHITE` instead of `white`

```suggestion
        return (Colour.values()[index]).toString().toLowerCase();
```